ui/packages/consul-ui/app/models/session.js

```diff
@@ -23,4 +24,9 @@ export default class Session extends Model {
   @nullValue([]) @attr({ defaultValue: () => [] }) ServiceChecks;
 
   @attr({ defaultValue: () => [] }) Resources; // []
+
+  @computed('NodeChecks', 'ServiceChecks')
+  get checks() {
+    return [...this.NodeChecks, ...this.ServiceChecks.map(item => item.ID)];
```

Comment:
Love array spreads.  If you want to get fancy and use destructuring too, I believe the function can be updated to `({ ID }) => ID`.  But this is stylistic and not required!

Reply:
Updated.